README.md

```diff
@@ -81,17 +81,19 @@ Note that the icons in this set are tied to be used in [Calypso](https://github.
 
 This icon set uses a few automation scripts to ease the generation of new icons in a reliable way. In short, we require `node` and `grunt`. For detailed instructions check [the installation page](https://github.com/Automattic/gridicons/wiki/Installation).
 
+Once you checkout the repo run `npm install` in the `gridicons` folder. 
+To generate all the fonts, svgs and so on you run `grunt` 
```

Comment:
`npm run build` no need for global grunt.

Reply:
https://github.com/Automattic/gridicons/wiki/Installation can also be updated to remove the global install

Reply:
Eg remove step 4, and update step 6 to `npm run build`